src/types.ts

```diff
@@ -47,6 +47,7 @@ export interface UserSession {
   authAppSecret?: string;
   authAppQrCodeUrl?: string;
   isPasswordChangeRequired?: boolean;
+  featureFlags?: any;
```

Comment:
Will these always be A/B testing feature flags? If yes, wondering if there's any naming confusion between this and the sort of feature flag that is set at app level like 'support international phone numbers'?